test/acceptance/kafo_export_params_test.rb

```diff
@@ -4,7 +4,10 @@ module Kafo
   describe 'kafo-export-params' do
     describe '--help' do
       it 'outputs usage' do
-        code, out, _ = run_command('bin/kafo-export-params --help', :dir => Dir.pwd)
+        generate_installer
```

Comment:
I think this should be a `before` block, similar to how it's done in `describe '-f'`. The `md` format can also benefit from it so you might be able to move it all the way to the top so every test in this file first generates an installer and adds a manifest.